ingest/v2.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Ingest API v2
+tags: overview
+description: Use the TelemetryDeck Ingest API to send signals to TelemetryDeck
+lead: Use the TelemetryDeck Ingest API to send signals to TelemetryDeck
+order: 0
+headerImage: /img/ingestv2.jpg
+---
+
+Usually you'll send signals to TelemetryDeck using one of our SDKs. However, if you're working with a language or framework that we don't have an SDK for, you can send signals directly to our Ingest API.
+
+## Ingest Endpoint
+
+Please send your signals as POST request to our ingestion server at `https://nom.telemetrydeck.com/v2/`, with the headers `Content-Type: application/json` and `charset=utf-8`.
+
+Here's an example in cURL:
+
+```bash
+curl -X POST "https://nom.telemetrydeck.com/v2/" \
+    -H "Content-Type: application/json; charset=utf-8" \
+    --data '[{"appID":"AAAA-BBBBBBBB-CCCC-DDDD",
+              "clientUser":"myClientUserHash",
+              "type":"Curl.Development.apiCall"}]' \
```

Comment:
Is `Curl.Development.apiCall` usually how users name their signal types? Perhaps we can replace it with something more accessible like "buttonClicked".